### ISSUE TITLE
Remove QGrid from Datahub and add Modin in Biology Hub

### ISSUE THREAD

**Issue by yuvipanda** [berkeley-dsep-infra/hubs#3621]
Per @ericvd-ucb, MCB32 still uses qgrid. However, qgrid is basically abandoned, and doesn't work with ipywidgets 8.x

https://github.com/berkeley-dsep-infra/datahub/pull/3620 adds it back, but we should have MCB32 move to something else.

**Comment by balajialg** [comment]
- [x] @ericvd-ucb to provide context about the use case!

**Comment by ericvd-ucb** [comment]
Some background - this is used by Prof Robin Ball 
Main repo is here  https://github.com/ds-modules/MCB-32
QGrid is called in a Python script external to notebooks
 https://github.com/ds-modules/MCB-32/blob/master/lab05/qgrid_bridge.py

The point is that student are introduced to tables, then they enter data into a table from a lab, then they use their own lab data to explore topics such as visualization on, regression, prediction

<img width="1111" alt="image" src="https://user-images.githubusercontent.com/30128480/187999106-3b366c8f-be4c-48db-92d9-19d760c97f6d.png">

**Comment by ericvd-ucb** [comment]
I think the solution might be https://github.com/QuantStack/ipysheet

**Comment by yuvipanda** [comment]
@ericvd-ucb yes that looks like a good replacement! Should I add it to datahub?

**Comment by ryanlovett** [comment]
@yuvipanda @ericvd-ucb There was also ipydatagrid which was presented at Bloomberg recently, https://github.com/bloomberg/ipydatagrid. (just fyi, I've no opinion)

**Comment by ericvd-ucb** [comment]
hmm that looks better
I guess we should work on rebuilding the notebooks and then... figure out which one is better and install that

**Comment by balajialg** [comment]
I see no harm in installing both ipysheet and ipydatagrid unless we run into package versioning hell!

**Comment by balajialg** [comment]
@elias326 did a detailed analysis for the instructor to recommend the ideal replacement for Qgrid. He has chosen [ipysheet](https://github.com/QuantStack/ipysheet) and Eric VD recommended ipydatagrid to replace QGrid. Our next step here is to figure out ways to install both packages in the biology hub image. 

I am currently waiting on approval from the instructor to move forward.

**Comment by balajialg** [comment]
- [x] Tag Peter when we have some clarity about the requirement?
- [x] Write an email to Robin Ball

**Comment by balajialg** [comment]
- [ ] Write to Robin Ball inquiring about the status of the package decision.

**Comment by balajialg** [comment]
Instructor needs to decide the package and then use it only during summer (by mid June)! So - scoping this request for May!

**Comment by balajialg** [comment]
Instructor confirmed that they are interested to use ipydatagrid package as a replacement for qgrid! We need to scope this for the month of May.

**Comment by balajialg** [comment]
Jonathan Ferrari recommends to use Modin package for this course. He will update this GitHub issue to install the package when the notebook is ready for deployment in the bio hub.

**Comment by ryanlovett** [comment]
https://blog.jupyter.org/ipydatagrid-is-now-part-of-project-jupyter-3b3dfb877664
> ipydatagrid is now part of Project Jupyter
